### DISCUSSION TITLE
debian rofi bug info

### DISCUSSION THREAD

**Discussion by kskarthik** [davatorium/rofi#1643] [Distro Specific] [answered]
I am maintaining rofi debian package, Please let me know if Is this issue still relevant in latest version of rofi ? https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=934609

also, please clarify on some wishlist items here https://bugs.debian.org/cgi-bin/pkgreport.cgi?repeatmerged=no&src=rofi

**Comment by DaveDavenport** [comment, marked as answer]
If the user find rofi useless he should stop using rofi. Its that simple.  The tone he uses makes me not want him as a user.

I have no intention or time to keep track of every distributions issue list, I strongly suggest automatically close those issues, and send them to this discussion forum.

The reason for this is simple:
1.) You nor me have time to deal with this.  For me there is no way I can keep track with the hundreds? of distroes.
2.) All except one are already resolved years ago. The one open is such a bad feature request I would not deal with it to start with.

**Reply by kskarthik** [reply]
thanks, noted
